src/docs/arch-rts.dox

```diff
@@ -0,0 +1,112 @@
+/*! @arch_page arch-rts Rollback to stable
+
+Rollback to stable is an operation that retains only the modifications that
+are stable according to the stable timestamp and recovered checkpoint snapshot.
+
+@section rts-overview Overview of rollback to stable
+
+Rollback to stable process scans each and every table present in the database
+except metadata @ref arch-metadata to remove the modifications from the table
+that are more than stable timestamp and recovered checkpoint snapshot.
+
+In the process of removing newer modifications from the table, all the in-memory
+updates are aborted and the on-disk version updates are replaced with an update
+from history store if exist. In case if there is no history store version is
```

Comment:
In case if there is no history store version available, the data store version is removed. ?

Reply:
Reworded the entire sentence. Check it now?